### PR TITLE
Rachana_Assignment_24_Done_25mint

### PR DESCRIPTION
**Pull request by RachanaGJun20**
image

### CODE REVIEW

File: src/rachanaGJun20/String/PhoneNumberValidation.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		}
+		if (flag)
+			return true;
+		else
+			return false;
+	}
+	
+	public static void main(String[] args) {
+		PhoneNumberValidation phonenumbervalidation = new PhoneNumberValidation();
+		String[] phNumber = { "+91 9765463742", "09765463742", "976 546 3742", "9765463742", "9765463A42", "976546342",
```
Comment (AjitSJune20) [inline src/rachanaGJun20/String/PhoneNumberValidation.java:63]: Can you check with input like this
+91 9765 463742
9765 46 3742
@RachanaGJun20   length and spacing of phonenumber not handled properly. 
  
@KrishnaKTechnocredits Plzz give your input.

Comment (RachanaGJun20) [reply src/rachanaGJun20/String/PhoneNumberValidation.java:63]: image


@AjitSJune20  Hey Ajit, Thanks for giving the another input numbers, my program is working for your given combination as well. Please have a look.

Comment (AjitSJune20) [reply src/rachanaGJun20/String/PhoneNumberValidation.java:63]: It should throw Invalid @RachanaGJun20 bcoz of exceeding length of PhoneNumber and space is at wrong place in Phonenumber

File: src/rachanaGJun20/String/PhoneNumberValidation.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			System.out.println("Number not valid");
+	}
+
+	// it will check that number does not contains any character
+	boolean isAllDigits(String phoneNumber) {
+
+		boolean flag = true;
+		for (int index = 1; index < phoneNumber.length(); index++) {
+			if (!Character.isDigit(phoneNumber.charAt(index))) {
+				flag = false;
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/String/PhoneNumberValidation.java:52]: break statement is missing here.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Going to merge it.

**Pull request merged**